content/en/docs/release-notes/release-notes-0.2.md

```diff
@@ -7,7 +7,7 @@ type: "docs"
 
 ## Changelog since v0.1.0
 ### Action Required
-- Move to 'jetstack' organisation. Action required: this will require updating your existing deployments to point to the new image repository, as new tags will not be pushed to the old 'jetstackexperimental/cert-manager-controllerrepository. Ahelm upgrade` should take care of this. (#145, @munnerz)
+- Move to 'jetstack' organization. Action required: this will require updating your existing deployments to point to the new image repository, as new tags will not be pushed to the old 'jetstackexperimental/cert-manager-controllerrepository. A helm upgrade` should take care of this. (#145, @munnerz)
```

Comment:
Not you, but there is a backtick here for no reason after the word 'upgrade' causing this to render weirdly